.gitignore

```diff
@@ -9,3 +9,4 @@ phpunit.xml
 .DS_Store
 Thumbs.db
 .nvmrc
+/yarn.lock
```

Comment:
npm would be the build tool used in this repo and we shouldn't be crossing streams so to speak.